### PR TITLE
add require to base event

### PR DESCRIPTION
**Pull request by addicted2sounds**
Getting `NameError: uninitialized constant Hermes::Dry`. fix for `dry-struct` requirement

### CODE REVIEW

**Comment by addicted2sounds** [comment]
nvm, I see it's fixed in the wrong place, need to figure out how to change this

File: lib/hermes/base_event.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+require 'dry-struct'
```
Comment (Azdaroth) [inline lib/hermes/base_event.rb]: please use double quotes

**Comment by Azdaroth** [comment]
@addicted2sounds Does it solve the issue in the end? Have you tested it?

**Comment by addicted2sounds** [comment]
@Azdaroth yes, the error I got initially was:
```
NameError: uninitialized constant Hermes::Dry
```
checked with requiring the local version of the gem. Not sure how to check that with specs

**Comment by Azdaroth** [comment]
@addicted2sounds Ok, that should be enough, all good then

**Review by Azdaroth — APPROVED** [review]
💪 🐉 Opa!

**Pull request merged**